metaconfig-core/shared/src/main/scala/metaconfig/internal/Priority.scala

```diff
@@ -0,0 +1,55 @@
+package metaconfig.internal
+
+/**
+  * Copied from https://github.com/monix/implicitbox/blob/master/shared/src/main/scala/implicitbox/Priority.scala
```

Comment:
Is there any legal issues with copying code from another library? Should I add license or smt?